packages/e2e-tests/specs/experiments/navigation.test.js

```diff
@@ -267,7 +267,7 @@ beforeEach( async () => {
 afterEach( async () => {
 	await setUpResponseMocking( [] );
 } );
-describe( 'Navigation', () => {
+describe.skip( 'Navigation', () => {
```

Comment:
Is there an Issue created for a follow up here? The code around this is fairly involved and quite easy to break. Would be good to retain some e2e test coverage.

I might be able to help this Friday if folks feel it's worthwhile.

Reply:
I don't know that there's a followup issue, but I think @mkaz meant to look at it soon!

Reply:
Tracking issue created here: https://github.com/WordPress/gutenberg/issues/27588